core/tests/performance_tests/test_config.py

```diff
@@ -32,17 +32,138 @@
 """
 
 SPLASH_PAGE_KEY = 'splash'
```

Comment:
Call these PAGE_KEY_SPLASH, PAGE_KEY_LIBRARY, PAGE_KEY_CREATOR_DASHBOARD, etc. Having a common prefix makes it clearer that these constants belong to the same "group".